syncthing.te

```diff
@@ -0,0 +1,45 @@
+policy_module(syncthing, 1.0.0)
+
+# define types
+type syncthing_t;
+type syncthing_exec_t;
+userdom_user_application_domain(syncthing_t, syncthing_exec_t)
+
+type syncthing_config_home_t;
+userdom_user_home_content(syncthing_config_home_t)
+
+# file permissions
+fs_getattr_xattr_fs(syncthing_t)
+
+userdom_manage_user_home_content(syncthing_t)
+
+# read randomness
+dev_read_rand(syncthing_t)
+dev_read_urand(syncthing_t)
+
+# networking
+userdom_basic_networking(syncthing_t)
+
```

Comment:
`userdom_basic_networking`? Might be downstream in Fedora/RedHat land :disappointed: 

Can you suggest an alternate?